applications/asset_tracker_v2/src/modules/debug_module.c

```diff
@@ -278,9 +292,20 @@ static void memfault_handle_event(struct debug_msg_data *msg)
 	 * compared to having Memfault SDK trigger regular updates independently. All data
 	 * should preferably be sent within the same LTE RRC connected window.
 	 */
-	if ((IS_EVENT(msg, data, DATA_EVT_DATA_SEND)) ||
-	    (IS_EVENT(msg, cloud, CLOUD_EVT_CONNECTED))) {
-		send_memfault_data();
+	if (IS_EVENT(msg, data, DATA_EVT_DATA_SEND)) {
+		send_memfault_data(false);
```

Comment:
Is it intentional that we don't send data after we get an LTE connection?

Reply:
We still send data. The passed in flag is only used to distinguish between data-sending on LTE connected vs on all other occasions.